src/utils/index.js

```diff
@@ -8,4 +8,8 @@ export const determinePropertyMessage = (actual, property, message = 'Not Access
   return actual && actual[property] ? actual[property] : message;
 };
 
+export const isJestMockOrSpy = obj => {
+  return !!obj._isMockFunction && typeof obj.mock !== 'undefined';
```

Comment:
I'm having trouble reading this logic 🙊, could you break it out to be more explicit so simple minds like mine can parse this easier please 😄

Reply:
Well, not sure how to break it. `_isMockFunction` is a boolean prop which jest mocks and spies have. Also since matchers use values from `mock` object, I added check for this. Probably `typeof obj.mock === "object"` would be more explicit.

And in case this  `!!` confuses you, it can be safely removed 😄  Just since I mostly work with typescript, I usually add this explicit boolean cast.

Reply:
Please check if it became better now

Reply:
Ugh, formatted it with line-breaks, but `prettier` knows better 😞